tests/integration_tests/test_sql_injection_protection.py

```diff
@@ -1,160 +1,200 @@
-            schema_json=json_schema,
-        ).get()
-
-
-def test_sql_injection_create_table_tableschema_name():
-    with pytest.raises(ValueError):
-        schema = TableSchema(
-            [
-                ColumnInfo("Robert'); DROP TABLE data; --", "INT"),
-                ColumnInfo("col2", "real"),
-                ColumnInfo("col3", "text"),
-            ]
-        )
-        json_schema = schema.to_json()
-        local_node_create_table.delay(
-            context_id=context_id,
-            command_id=str(uuid.uuid1()).replace("-", ""),
-            schema_json=json_schema,
-        ).get()
-
-
-def test_sql_injection_get_merge_tables():
-    with pytest.raises(ValueError):
-        local_node_get_merge_tables.delay(
-            context_id="Robert'); DROP TABLE data; --"
-        ).get()
-
-
-def test_sql_injection_create_merge_table_table_names():
-    with pytest.raises(ValueError):
-        local_node_create_merge_table.delay(
-            context_id=context_id,
-            command_id=str(uuid.uuid1()).replace("-", ""),
-            table_names=["Robert'); DROP TABLE data; --"],
-        ).get()
-
-
-def test_sql_injection_get_views():
-    with pytest.raises(ValueError):
-        local_node_get_views.delay(context_id="Robert'); DROP TABLE data; --").get()
-
-
-def test_sql_injection_get_view_schema():
-    with pytest.raises(ValueError):
-        local_node_get_table_schema.delay(
-            table_name="Robert'); DROP TABLE data; --"
-        ).get()
-
-
-def test_sql_injection_create_view_columns():
-    with pytest.raises(ValueError):
-        columns = [
-            "Robert'); DROP TABLE data; --",
-            "age_value",
-            "gcs_motor_response_scale",
-            "pupil_reactivity_right_eye_result",
-        ]
-        datasets = ["edsd"]
-        pathology = "tbi"
-        local_node_create_pathology_view.delay(
-            context_id=context_id,
-            command_id=str(uuid.uuid1()).replace("-", ""),
-            pathology=pathology,
-            datasets=datasets,
-            columns=columns,
-            filters_json="filters_json",
-        ).get()
-
-
-def test_sql_injection_create_view_datasets():
-    with pytest.raises(ValueError):
-        columns = [
-            "dataset",
-            "age_value",
-            "gcs_motor_response_scale",
-            "pupil_reactivity_right_eye_result",
-        ]
-        datasets = ["Robert'); DROP TABLE data; --"]
-        pathology = "tbi"
-        local_node_create_pathology_view.delay(
-            context_id=context_id,
-            command_id=str(uuid.uuid1()).replace("-", ""),
-            pathology=pathology,
-            datasets=datasets,
-            columns=columns,
-            filters_json="",
-        ).get()
-
-
-def test_sql_injection_get_remote_tables():
-    with pytest.raises(ValueError):
-        local_node_get_remote_tables.delay(
-            context_id="Robert'); DROP TABLE data; --"
-        ).get()
+# import json
```

Comment:
From the call we had, I understood that this will be deleted, since these tests cannot pass, due to filters being added and a jira task will be created to add sql injection tests when pydantic is added.